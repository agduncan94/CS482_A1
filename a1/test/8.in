>header seq1
ACTGGTGATACATAGA
    ;  AAAAAAAATTTTTAAAAA
ACTAT
