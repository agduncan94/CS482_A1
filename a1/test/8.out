AAA 0.0000
AAC 0.0526
AAG 0.0000
AAT 0.0000
ACA 0.0526
ACC 0.0000
ACG 0.0000
ACT 0.1053
AGA 0.0526
AGC 0.0000
AGG 0.0000
AGT 0.0000
ATA 0.1053
ATC 0.0000
ATG 0.0000
ATT 0.0000
CAA 0.0000
CAC 0.0000
CAG 0.0000
CAT 0.0526
CCA 0.0000
CCC 0.0000
CCG 0.0000
CCT 0.0000
CGA 0.0000
CGC 0.0000
CGG 0.0000
CGT 0.0000
CTA 0.0526
CTC 0.0000
CTG 0.0526
CTT 0.0000
GAA 0.0526
GAC 0.0000
GAG 0.0000
GAT 0.0526
GCA 0.0000
GCC 0.0000
GCG 0.0000
GCT 0.0000
GGA 0.0000
GGC 0.0000
GGG 0.0000
GGT 0.0526
GTA 0.0000
GTC 0.0000
GTG 0.0526
GTT 0.0000
TAA 0.0000
TAC 0.0526
TAG 0.0526
TAT 0.0526
TCA 0.0000
TCC 0.0000
TCG 0.0000
TCT 0.0000
TGA 0.0526
TGC 0.0000
TGG 0.0526
TGT 0.0000
TTA 0.0000
TTC 0.0000
TTG 0.0000
TTT 0.0000
